client/rest-high-level/src/test/java/org/elasticsearch/client/MachineLearningIT.java

```diff
@@ -435,7 +435,11 @@ public void testForecastJob() throws Exception {
             builder.addDoc(hashMap);
         }
         PostDataRequest postDataRequest = new PostDataRequest(jobId, builder);
-        machineLearningClient.postData(postDataRequest, RequestOptions.DEFAULT);
+        // Post data is deprecated, so expect a deprecation warning
+        RequestOptions requestOptions = RequestOptions.DEFAULT.toBuilder()
```

Comment:
Would it make sense to introduce `POST_DATA_OPTIONS` static member just like you did in the other test?